api/versionedproposal.go

```diff
@@ -509,23 +509,45 @@ func (v *VersionedProposal) Timestamp() (uint64, error) {
 	}
 }
 
-// BlobSidecars returns the blob sidecars of the proposal.
-func (v *VersionedProposal) BlobSidecars() ([]*deneb.BlobSidecar, error) {
+// Blobs returns the blobs of the proposal.
+func (v *VersionedProposal) Blobs() ([]deneb.Blob, error) {
 	switch v.Version {
 	case spec.DataVersionPhase0:
-		return make([]*deneb.BlobSidecar, 0), nil
+		return make([]deneb.Blob, 0), nil
```

Comment:
this should return an unsupported version error? or at least nil for the array returned

Reply:
Updated to return an unsupported version error.